WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -248,7 +242,9 @@ private BobClient CreateBobClient(RoundState roundState)
 	private bool SanityCheck(IEnumerable<TxOut> expectedOutputs, Transaction unsignedCoinJoinTransaction)
 	{
 		var coinJoinOutputs = unsignedCoinJoinTransaction.Outputs.Select(o => (o.Value, o.ScriptPubKey));
-		var expectedOutputTuples = expectedOutputs.Select(o => (o.Value, o.ScriptPubKey));
+		var expectedOutputTuples = expectedOutputs
+			.GroupBy(x => x.ScriptPubKey)
+			.Select(o => (o.Select(x => x.Value).Sum(), o.Key));
```

Comment:
Well, in that case we would be preventing a user to be alone in a coinjoin what I am not sure it okay, i mean, nobody knows the only user participating is really only one and then by failing to sign it would be revealing that it is the same entity that controls all the inputs and all the outputs resulting in the deanonymation.
For that reason I think it is okay as it is.

Reply:
So my original thinking was that there are typically multiple users participating in the coinjoin transaction and even if there aren't I thought that there would be also a coordinator fee output - that would be more persuasive argument if it is true.

Thanks for the explanation.